src/Components/Impact.js

```diff
@@ -33,6 +33,50 @@ export default class Impact extends Component {
                 para2: "right way!",
               }}
             />
+          </Row>
+          <Row
+            className="justify-content-md-center"
+            style={{ marginTop: "5px" }}
```

Comment:
@Meghag5 Please try to avoid inline styling. It's not a good practice as it will be a problem when we scale our system.